src/components/scripture/dto/scripture-reference.transformer.ts

```diff
@@ -0,0 +1,31 @@
+import { applyDecorators } from '@nestjs/common';
+import { Transform } from '../../../common/transform.decorator';
+import { books } from '../books';
+import { bookIndexFromName } from '../reference';
+import { ScriptureReferenceInput } from './scripture-reference.dto';
+
+export const ScriptureStart = () =>
+  applyDecorators(
+    Transform((value: ScriptureReferenceInput) => {
+      value.chapter = value.chapter == null ? 1 : value.chapter;
+      value.verse = value.verse == null ? 1 : value.verse;
+      return value;
+    })
```

Comment:
Does this come before or after validation? We don't want validation to fail because the transform hasn't happened yet.

Reply:
Yeah, that was my concern.
I tested it and could see it comes before validation. So it works fine.